src/js/views/patients/sidebar/action/action-sidebar-action_views.js

```diff
@@ -130,12 +130,16 @@ const ActionView = View.extend({
     duration: '[data-duration-region]',
   },
   modelEvents: {
+    'change:details': 'onChangeDetails',
```

Comment:
We'll need to add something similar for the action name here in the sidebar.

Currently the action name won't be updated in the sidebar if a `NameChanged` ws message comes in.

Reply:
I actually think not doing this is a bug... but I don't think action names change much.  We have to handle events in the entire context of a page.

Reply:
Okay, I can tackle this in a separate story/PR.